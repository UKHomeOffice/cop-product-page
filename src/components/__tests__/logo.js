import React from "react";
import renderer from "react-test-renderer";
import Logo from "../logo";

describe("Logo", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Logo />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it("Returns the correct alt and source with its correct values given", () => {
    const tree = renderer.create(
      <Logo source={"http://test.com"} alt={"description"} />
    );
    const testInstance = tree.root;
    expect(testInstance.props).toHaveProperty("alt", "description");
    expect(testInstance.props).toHaveProperty("source", "http://test.com");
  });
});
