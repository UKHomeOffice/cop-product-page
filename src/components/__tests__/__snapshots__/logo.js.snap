// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo renders correctly 1`] = `
<img
  alt="logo"
  className="cop-logo"
/>
`;
